app/controllers/WorkerController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace GanttDashboard\App\Controllers;
+
+use GanttDashboard\App\Services\Worker as WorkerService;
+
+class WorkerController extends ControllerBase
+{
+    /**
+     * @var WorkerService
+     */
+    private $workerService;
+
+    /**
+     * Initialises the workerService property
+     */
+    public function onConstruct()
+    {
+        $this->workerService = $this->getDi()->get(WorkerService::class);
+    }
+
+    /**
+     * Performs the login
+     * @param string $accessKey
+     */
+    public function loginAction(string $accessKey = '')
+    {
+        $redirect = ['for' => 'notFound'];
```

Comment:
is very ugly to define not found pages by default, and overwrite them.
I know is a general practice to start with the bad/negation things first and then when code happens to assign a positive value.
But here is not the case... we would expect in this case to be a success... bun in case is not we want to mask it as the page is not there... we don't have that page.
I would say to use 2 redirects here....

Reply:
It won't work with 2 redirects.... It shows blank white page and the url remains unchanged. It works only like this or with dispatcher->forward

Reply:
I assure you that it works. There is no suck thing in programming: "it does not work"

Reply:
I talked with Flaviu. I'll use dispatcher->forward